Symfony/CS/Fixer.php

```diff
@@ -187,22 +187,27 @@ public function fixFile(\SplFileInfo $file, array $fixers, $dryRun, $diff, FileC
             return;
         }
 
+        $old = file_get_contents($file->getRealpath());
         $appliedFixers = array();
 
         // we do not need Tokens to still caching previously fixed file - so clear the cache
         Tokens::clearCache();
```

Comment:
hmm, actually, it seems we might still benefit from the cache for fixers setting the whole content again